monitoring/views/bigquery_usage.view.lkml

```diff
@@ -18,6 +18,16 @@ view: +bigquery_usage {
     type: string
   }
 
+  dimension: destination_table_id {
+    sql: split(destination_table_id, '$')[SAFE_OFFSET(0)];;
+    type: string
+  }
+
+  dimension: partition {
```

Comment:
Should we maybe also make this a date field? Could process the extracted value with `PARSE_DATE` and that would allow us to do `partition` field date-based filtering. Here's an example of what I mean:

https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_auth_events_v1/query.sql#L32

Reply:
It looks like not all partitions are the same format.  I will leave this field as string format until there is a need to adjust it.